src/health_monitor.rs

```diff
@@ -21,6 +21,8 @@ impl HealthMonitor {
     }
 }
 
+const CHAIN_STALE_IF_OLDER_THAN: std::time::Duration = std::time::Duration::from_secs(5 * 60);
```

Comment:
Michael suggested 5 minutes (see #205), but maybe we should fail when it’s less than that?